crowbar_framework/app/helpers/barclamp/neutron_helper.rb

```diff
@@ -17,35 +17,27 @@
 
 module Barclamp
   module NeutronHelper
-    def networking_plugins_for_neutron(selected)
-      options_for_select(
-        [
-          ["ml2", "ml2"],
-          ["vmware", "vmware"]
-        ],
-        selected.to_s
-      )
-    end
 
-    def networking_ml2_mechanism_drivers_for_neutron(selected)
-      selected = selected.gsub(/\s+/, "").split(",")
-      options_for_select(
-        [
-          ["linuxbridge", "linuxbridge"],
-          ["openvswitch", "openvswitch"],
-          ["cisco_nexus", "cisco_nexus"],
-        ],
-        selected
-      )
+    def self.networking_plugins_valid()
+      ["ml2", "vmware"]
     end
 
-    def networking_ml2_type_drivers_valid()
+    def self.networking_ml2_type_drivers_valid()
```

Comment:
If there are no parameters remove the ```()``` please. Anyway validation stuff should be placed in the model.